python/ray/tune/examples/README.rst

```diff
@@ -53,6 +53,13 @@ TensorFlow Examples
    Example of tuning a TensorFlow model on MNIST using AsyncHyperBand.
 
 
+MXNet Gluon Example
```

Comment:
Thanks for the contribution! Can you please put this under `Contributed Examples`? 

Feel free to add an email address so that people can contact you if this file fails, as we will not be testing this in our CI.

Reply:
Done, thanks.